client/homebrew/pages/accountPage/accountPage.jsx

```diff
@@ -29,6 +31,32 @@ const AccountPage = createClass({
 			uiItems : this.props.uiItems
 		};
 	},
+	componentDidMount : function(){
+		if(!this.state.saveLocation && this.props.uiItems.username) {
+			SAVEKEY = `HOMEBREWERY-DEFAULT-SAVE-LOCATION-${this.props.uiItems.username}`;
+			let saveLocation =  window.localStorage.getItem(SAVEKEY) || '';
+			if(saveLocation == '') {
+				saveLocation = 'HOMEBREWERY';
+				if(this.state.uiItems.googleId) {
+					saveLocation = 'GOOGLE-DRIVE';
+				}
+			};
+			this.makeActive(saveLocation);
+		}
+	},
+
+	makeActive : function(newSelection){
+		if(this.state.saveLocation == newSelection) return;
+		window.localStorage.setItem(SAVEKEY, newSelection);
+		this.setState({
+			saveLocation : newSelection
+		});
+	},
+
+	renderButton : function(name, key, condition=true){
```

Comment:
Can we rename "condition" to something more descriptive? Like "shouldRender" or something?

Reply:
Applied in commit [7feaa51](https://github.com/naturalcrit/homebrewery/pull/2944/commits/7feaa51de081957dfa6730f11dafe1811cd78dbd)